### DISCUSSION TITLE
Can it be realize “crosshair”？

### DISCUSSION THREAD

**Discussion by EXHades** [JetBrains/ideavim#285] [Q&A] [answered]
vim:
```
set cursorline
set cursorcolumn
```
![image](https://user-images.githubusercontent.com/22260242/114383234-e94b0a00-9bbf-11eb-83fb-eba9b0fe0ffc.png)

But not work in IDEA？

**Comment by AlexPl292** [comment, marked as answer]
Hi! IJ highlights the caret line by default. `cursorcolum` option is not supported and we don't have any plans regarding it at the moment.

**Reply by EXHades** [reply]
Ok, I get it. Thank you.

**Comment by est7** [comment]
@AlexPl292 hello, could I set cursorLine highlight color?

**Reply by AlexPl292** [reply]
You can change it in this setting
<img width="1270" alt="image" src="https://github.com/JetBrains/ideavim/assets/4203721/815d268f-6282-4144-ad79-6f54cca9f091">
